.factory/automation.yml

```diff
@@ -56,71 +56,73 @@ build:
     deploy-crate-snapshot:
       filter:
         owner: typedb
-        branch: [master, development]
+        branch: [master, development, "3.0"]
       image: typedb-ubuntu-22.04
-      dependencies: [build, build-dependency, test-rust]
+      dependencies: [build, build-dependency, test-rust-unit]
```

Comment:
Let's at least put a todo to add the test-rust-behaviour dependency when it's fixed